src/lib/features/Card.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  import { UserFilled } from '/src/lib/assets/index';
+
+  export let title = 'Default Card Header';
+  export let bgColor = 'cream';
+  export let color = 'black'
+  export let titleColor = 'navy'
+  export let icon = UserFilled;
+</script>
+
+<div class="p-6 card-shadow rounded-xl bg-{bgColor} {$$restProps.class}">
```

Comment:
```suggestion
<div class="p-6 card-shadow rounded-xl bg-cream {$$restProps.class}">
```
Thinking we degault to cream, then when a user wants to change the color, they can just set the class to the new color that they want and it can override the cream through the restProps.class.

Reply:
I did it this way because the background color as a prop was part of the Issue specification.  It does already default to cream, however, I think it's not the best idea to have two background color classes end up in the class list.